src/server/pfs/server/server_test.go

```diff
@@ -1704,22 +1723,6 @@ func TestPutFileWithNoDelimiter(t *testing.T) {
 
 }
 
-func TestPutFileNullCharacter(t *testing.T) {
```

Comment:
Pretty sure we still want this test, the issue with Null characters wasn't that our backend doesn't handle them. It's that anything written in C (like `ls`) gets tripped up because if we return a file called `ihavea\0characterinmyname` it interprets that as just `ihavea` which the user then tries to read and gets a file not found error.

Would you mind adding a comment to this test explaining the reasoning behind it so there's no confusion in the future?